protostar/commands/cairo1_commands/test_rust_command.py

```diff
@@ -33,8 +40,18 @@ def arguments(self):
         ]
 
     async def run(self, args: Namespace):
+        contract_paths: dict[str, list[str]] = {}
```

Comment:
```suggestion
        contract_paths: Dict[str, List[str]] = {}
```

I think it's better to use `Dict` and `List` from typing package here.

Reply:
I don't think this got resolved